Update i_love_how_animals_taste.md (#34)
veganhacktivists/animalsupportbot: knowledge/responses/i_love_how_animals_taste.md

When it boils down to it, the reason people eat animal products is because they enjoy the taste - in fact, most vegans loved the taste of animal products before going vegan. But sensory pleasure is a poor excuse. It bluntly admits that an individual’s taste preference matter more than the morality surrounding an animal’s life and unquestionably horrific death. 

> The question is: “do you value your taste buds higher than the life of an animal”? And it is not only about the taking of the animal's life. To reach your plate and give you a few minutes of pleasure, animals are forcefully in-bred, live horrendous lives of constant abuse, inhabit bodies that no longer work properly due to genetical modification, and are finally taken to death camps where they see their peers being killed one by one. This is all in accordance with "humane" regulations.

Reviewer: Though this may seem obvious and well-known to us, I would suggest adding a source that describes the horrendous conditions faced by farmed animals.

> So, if you think that a few minutes of pleasure justifies the immense cruelty these sentient beings are made to suffer, you'd need to think what else can be justified by pleasure alone. Would this argument hold up in court if a murderer in their own defence said, “but I could never give up murdering because I enjoy it too much.”? For instance, in places in the world where women are second class citizens, the way they dress is considered a justification for rape. They are seen as having incited the will for physical pleasure in their assailants.

Reviewer: ```suggestion
If you think that a few minutes of pleasure justifies the immense cruelty these sentient beings are made to suffer, you'd need to think what else can be justified by pleasure alone. Would this argument hold up in court if a murderer in their own defense said that they could never give up murdering because of pleasure? 
```

Do you think taste justifies the shark fin trade in Asia? The turtle flesh market in northern South America? The dog meat festival in South Korea? The dolphin and whale slaughters in Japan?

> It requires more than sensory pleasure to morally justify something - and the inconvenience people go through by giving up cheese, for example, is not even remotely comparable to the pain a dairy cow goes through being forcibly impregnated repeatedly, having her children taken away from her and being painfully pumped for her milk before she becomes too weak and is sent off to be slaughtered. 

Reviewer: Add source.

> It doesn’t matter whatever product people claim they could never give up or enjoy the taste of. The difficulty and the inconvenience that it would be for them to no longer eat animal products is minuscule compared to the pain, suffering and fear that the animals have to go through. A non-vegan meal lasts for 15 minutes, but the abuse of an animal lasts through their lives, and their death is irreversible. What food could ever be worth taking the life of an individual? An individual that felt fear, an individual that felt pain, an individual that felt confused and couldn’t understand the intolerable suffering that was being committed against them. The scariest thing we can think of is being tortured and ultimately murdered. This is the reality the trillions of animals that are slaughtered each year, it isn’t some nightmare that they will wake from. It’s real, the pain, the fear, it is all real.

Reviewer: ```suggestion
It doesn’t matter whatever product people claim they could never give up or enjoy the taste of. The difficulty and the inconvenience that it would be for them to no longer eat animal products is minuscule compared to the pain, suffering and fear that the animals have to go through. A meal may last for 15 minutes, but the abuse of an animal lasts through their lives and their death is irreversible. What food could ever be worth taking the life of an individual? An individual who felt fear, pain, and confusion due to the intolerable suffering being committed against them. The scariest thing we can think of is being tortured and ultimately murdered. Yet, this is the reality of trillions of animals that are slaughtered each year.
```

> It doesn’t matter whatever product people claim they could never give up or enjoy the taste of. The difficulty and the inconvenience that it would be for them to no longer eat animal products is minuscule compared to the pain, suffering and fear that the animals have to go through. A non-vegan meal lasts for 15 minutes, but the abuse of an animal lasts through their lives, and their death is irreversible. What food could ever be worth taking the life of an individual? An individual that felt fear, an individual that felt pain, an individual that felt confused and couldn’t understand the intolerable suffering that was being committed against them. The scariest thing we can think of is being tortured and ultimately murdered. This is the reality the trillions of animals that are slaughtered each year, it isn’t some nightmare that they will wake from. It’s real, the pain, the fear, it is all real.

Reviewer: I would add source to "trillions of animals that are slaughtered each year"
